_data/sidebars/lb4_sidebar.yml

```diff
@@ -98,6 +98,15 @@ children:
   url: Crafting-LoopBack-4.html
   output: 'web, pdf'
 
+- title: 'Language Related Concepts'
```

Comment:
Is it worth starting a new top-level section instead of nesting this new "Mixin" page under "Key concepts"? I don't know what's the right answer, just something to consider.

Reply:
@bajtos I feel at least now `Mixin` is not a LB core concept, we don't have mixin level context, and don't reply on it to scaffold the app, it is more like a JS/TS strategy to extend class, which make other concepts extendable, like application, controller, as long as it's a class.

there is some context of this discussion, I pinged Raymond about moving `mixin.ts` from `@loopback/repository` to `@loopback/core`, and he suggests we can have a `@loopback/lang` or `@loopback/common` module for JS/TS related stuff.

I created an issue for this idea: https://github.com/strongloop/loopback-next/issues/660

Reply:
S/be 
Language-related concepts